Expensify/App
# [TS migration] Migrate 'BigNumberPad.js' component to TypeScript

### Details



### Fixed Issues

$ https://github.com/Expensify/App/issues/25032
PROPOSAL: N/A


### Tests

1. Open the Native and Browser App on a mobile device.
2. Request money and go to Manual tab.
3. Enter some amount, use backspace, ensure everything is working.
4. Move to next page and ensure everything is working.

### Offline tests

Same as above.

### QA Steps

Same as above.

### PR Author Checklist


### Screenshots/Videos

<details>
<summary>Android: Native</summary>


https://github.com/Expensify/App/assets/20051562/48fa7603-60e6-4a2e-a782-406714287827



</details>

<details>
<summary>Android: mWeb Chrome</summary>


https://github.com/Expensify/App/assets/20051562/0b1e4e46-722c-47a1-a753-f11a938b1238


</details>

<details>
<summary>iOS: Native</summary>


https://github.com/Expensify/App/assets/20051562/7b2d8293-675a-4dfd-b932-a2275f0e04ce



</details>

<details>
<summary>iOS: mWeb Safari</summary>


https://github.com/Expensify/App/assets/20051562/24531051-2ba7-4cfe-8201-a127ead14ea8



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>
